canonical-web-and-design/maas-ui
# fix(ui): use correct unit for NUMA memory

## Done

- Fixed NUMA memory which is reported as B, not MiB like in general memory.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the NUMA view of bolla and check that the total memory matches the physical memory of the pod (i.e. at overcommit = 1)

## Fixes

Fixes #1689
